### DISCUSSION TITLE
Mobile App for the Admin UI

### DISCUSSION THREAD

**Discussion by DennisKraaijeveld** [swellstores/community#132] [Ideas]
Is there a mobile app for the admin UI on the roadmap? So we can receive notifications when products are sold etc etc. And it will allow to make some quick changes to products

**Comment by swellmike** [comment]
We can pass this request to the product team, but as of now, there are no immediate plans for a Swell Admin - mobile app.

As for notifications when a product is sold, you should be able to do this today via order webhooks.  Depending on how customized the notifications logic needs to be, you may even be able to configure this out-of-the-box by browsing to Settings > Notifications within the Admin UI.

**Comment by ericingram** [comment]
@aaronlee777 started prototyping this a year or so ago, but we had to pause it for other priorities. It will be great when it happens!
